### ISSUE TITLE
badge: <language> for <your name>

### ISSUE THREAD

**Issue by Kasahe** [revoltchat/translations#140]
### What language(s) did you translate?

French

### What is your Revolt ID?

01GNPRBTDX6QCJ31RNDCZKFQNJ

### Validations

- [X] The language has a maintainer, [see list](https://github.com/revoltchat/translations#language-maintainers).
- [X] You agree that your translations are to a reasonable standard.

**Comment by Hades785** [comment]
Hello,

You have made a bunch of suggestions, which I appreciate even though I rejected most of them in the end…
For a good part of them it was because they didn't feel right (well, most of them I had explained to you already about the non-breaking space thing), but some of them weren't correct in relation to their context (correct translation in other contexts but not quite in context of where they're used in the app). Although you did catch a few things that I wouldn't have caught otherwise (or not until much later), like the error in "Fa**î**tes" and one string translating "click" as "cli**que**" instead of "cli**c**".

There were a couple of cases where I'd have wanted to tell you to read comments on existing translations (Like "bot" translated to "bot" (English borrow) rather than "robot" (French translation): a note had been made on one string. Same for "upload" → "téléversement").
And another one yesterday had me quite confused with you translating "deafen" as "rendre muet" (since "deaf" → "sourd", not "muet")…

So a lot of your contributions weren't kept, but those that were were things I would have taken a long time to catch… It's a funny situation. But that's how localisation works, there aren't always good 1:1 translations so I'm grateful for suggestions, even if they aren't accepted. (though a part of French people seem to always want upload ⇒ téléchargement and bot ⇒ robot…)
So yeah, although your contributions weren't very complex, they were (for the ones I kept) nice to have.
_I would not be opposed to you getting recognition for them, but they just aren't very numerous. **I don't know whether I should agree or not because of that.**_ I would like to see more contributions from you beforehand, but at the same time there should not be a lot left to do any more so it's harder to get more contributions.

Also there's the very last suggestion which is okay in that it's correct context-wise but completely ignores the English base string, and I just don't know what to do about it because I find the English base lacking but at the same time your suggestion is quite verbose (context: server member masquerading permission `permissions.Masquerade.t`).

**Comment by Kasahe** [comment]
Hello,

For the translation Masquerade it should really be changed in France we
don't use this kind of word and I find that the translation I provided is
more complete and professional.

Sincerely, Kasahe

**Comment by Kasahe** [comment]
I helped again with the translation this morning because I had some time, I
really plan to invest myself on Revolt because I love the application I
recently got the agreement to get my server verified that's why I'm so
involved thank you for your understanding.

**Comment by Hades785** [comment]
> For the translation Masquerade it should really be changed

I really do agree on that. I'm just not sure we should have something that long. For reference, here's a screenshot of that screen: 
![image](https://user-images.githubusercontent.com/30666056/226177822-d5424ef1-f153-43c0-bdfa-80b3ccfd97ce.png)

> I helped again with the translation this morning because I had some time, I
really plan to invest myself on Revolt because I love the application I
recently got the agreement to get my server verified that's why I'm so
involved

Thank you!
It's great to have people involved into making the app better in any way they can contribute. The translation is a community effort after all, I'm more than happy to have more people working on it.

(I'll take a look at the rest later on, I have to go drive home. But at least if you really want to get involved, then I'll be fine with you having the badge. And it will let me discuss the translation work with someone, if you're fine with it. Haven't been able to do that in a year now =w=)

**Comment by Kasahe** [comment]
>
> Thank you for responding,

I have also translated more than 30% for Armenian language I would like to
have the reward of course I will continue and if you want to give me your
nickname to exchange directly on revolt.

Sincerely, Kasahe

>
>

**Comment by Hades785** [comment]
> I have also translated more than 30% for Armenian language

Ah, well I didn't check for that, I'm not maintaining the Armenian localisation =w=
Armenian doesn't look to have a maintainer so you could ask for maintainership.

**Comment by Hades785** [comment]
Alright @Rexogamer, you can mark them as translator for French, now. Thank you.

And thank you @Kasahe again for your contributions!

**Comment by Rexogamer** [comment]
Fixed in 6fd1006cc4e7811688c9b2f8313a4a833998f6fa
